### PR TITLE
fix: deny create table with distribution options for standalone DB

### LINKED ISSUE #621: Deny create table with distribution options for standalone DB
Issue tracker is **ONLY** used for reporting bugs. New features should be discussed on our discussion

Now, we have the standalone DB. 
It might be inappropriate if we support creating a table with distribution and replica configuration.
We have to give a warning or just deny creating a table when users configure cluster options.
## Expected Behavior

```cmd
create table {auto} (
      c1 string, c3 int, c4 bigint, c5 float, c6 double, c7 timestamp, c8 date,
      index(key=(c3), ts=c4))
      options (
      replicanum = 3,
      partitionnum = 8,
      distribution = [ ('{tb_endpoint_0}', ['{tb_endpoint_1}'])]
      )
```

ERROR: Fail to create a table with the distribution configuration in standalone DB
## Steps to Reproduce

1.
2.
3.
4.

### PR DESCRIPTION
**Pull request by keyu813**
close #621

### CODE REVIEW

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -1377,6 +1377,18 @@ base::Status SQLClusterRouter::HandleSQLCreateTable(hybridse::node::CreatePlanNo
     }
     ::openmldb::nameserver::TableInfo table_info;
     table_info.set_db(db_name);
+
+    if (!cluster_sdk_->IsClusterMode()) {
+        if (create_node->GetReplicaNum() != 1) {
+            return base::Status(base::ReturnCode::kSQLCmdRunError, 
+            "ERROR: Fail to create table with the replica configuration in standalone DB");
```
Comment (aceforeverd) [inline src/sdk/sql_cluster_router.cc]: indent. u can format changes with clang-format.
Also fix those cpplint warnings

Comment (aceforeverd) [inline src/sdk/sql_cluster_router.cc:1384]: indent.
u can format changes with clang-format

Comment (keyu813) [reply src/sdk/sql_cluster_router.cc]: > indent. u can format changes with clang-format. Also fix those cpplint warnings

ok I will update the format

**Review by vagetablechicken — APPROVED** [review]

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -1377,6 +1375,18 @@ base::Status SQLClusterRouter::HandleSQLCreateTable(hybridse::node::CreatePlanNo
...
     table_info.set_db(db_name);
+
+    if (!cluster_sdk_->IsClusterMode()) {
+        if (create_node->GetReplicaNum() != 1) {
+            return base::Status(base::ReturnCode::kSQLCmdRunError,
+                                "ERROR: Fail to create table with the replica configuration in standalone DB");
+        }
+        if (!create_node->GetDistributionList().empty()) {
+            return base::Status(base::ReturnCode::kSQLCmdRunError,
+                                "ERROR: Fail to create table with the distribution configuration in standalone DB");
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: in standalone mode

**Review by dl239 — APPROVED** [review]

**Pull request merged**